services/rfq/relayer/relconfig/getters.go

```diff
@@ -808,3 +808,28 @@ func (c Config) GetQuoteSubmissionTimeout() time.Duration {
 	}
 	return timeout
 }
+
+var defaultVolumeLimit *big.Int
```

Comment:
@golangisfun123  sorry is this supposed to be nil by default?

Right now this will error if unset here: https://github.com/synapsecns/sanguine/blob/aba64a79c79d30a8502a1213274d893f75b0124d/services/rfq/relayer/relconfig/getters.go#L818

It's not even clear how this is supposed to work. If we expect 0 by default, then all txes are over volume by default since this will always return false

https://github.com/synapsecns/sanguine/blob/aba64a79c79d30a8502a1213274d893f75b0124d/services/rfq/relayer/limiter/limiter.go#L127

Reply:
fixed, added a check. 0 is a no-limit value. 
`var defaultVolumeLimit *big.Int = common.Big0`

```
	volumeLimit := l.cfg.GetVolumeLimit(int(request.Transaction.OriginChainId), request.Transaction.OriginToken)
	// There is no limit.
	if volumeLimit.Cmp(common.Big0) == 0 {
		return true, nil
	}
```

Reply:
we can also do a nil check instead, what do you think is better?

Reply:
I'd do -1 actually. We might want to do 0 to disable, so I'd default to -1 

btw, b/c big ints are pointers I generally discourage the use of common.Big0 type stuff since these  can be mutated as a side effect, especially by a bug in another lib dependent on 

```go
common.Big0 = big.New(5)
```

we also copy any globals to prevent accidental mutation.